README.md

```diff
@@ -1,4 +1,4 @@
-# microformats-ruby
+# <span role="presentation" style="display: inline-block; vertical-align: middle; width: 1em">![''](https://raw.githubusercontent.com/indieweb/microformats-ruby/master/logo.svg?sanitize=true)</span> microformats-ruby
```

Comment:
This URL should work once this is merged to `master`. The `?sanitize=true` param on the end of the URL is a flag to GitHub to serve this image as an `image/svg+xml` instead of plaintext.